cumulusci/tasks/datadictionary.py

```diff
@@ -28,44 +54,139 @@ class GenerateDataDictionary(BaseGithubTask):
         "field_path": {
             "description": "Path to a CSV file to contain an field-level data dictionary."
         },
-        "release_prefix": {
-            "description": "The tag prefix used for releases.",
-            "required": True,
+        "include_dependencies": {
+            "description": "Process all of the GitHub dependencies of this project and include their schema in the data dictionary.",
```

Comment:
Really tempted to say schemata here but I guess it won't help comprehension...

Reply:
I love it. I think it's okay stylistically to use schema as an uncounted noun here but I will never pass up a chance at a third-decension plural.